src/views/Player.vue

```diff
@@ -81,11 +116,114 @@ export default defineComponent({
 
     getQuiz();
 
+    /**
+     * defines all the metrics to show in the scorecard here
+     */
+    const scorecardMetrics = computed(() => {
+      return [
+        {
+          name: "Correct",
+          icon: {
+            source: "correct",
+            class:
+              "text-[#10B981] h-7 bp-360:h-8 bp-500:h-10 lg:h-11 w-8 bp-360:w-8 bp-500:w-10 md:w-10 mt-5 bp-360:mt-6 md:mt-4 lg:mt-5 my-1 lg:w-11 place-self-center",
+          },
+          value: state.numCorrect,
+        },
+        {
+          name: "Wrong",
+          icon: {
+            source: "wrong",
+            class:
+              "text-red-500 h-8 bp-360:h-8 bp-500:h-10 md:h-11 w-6 bp-360:w-6 bp-500:w-6 md:w-7 lg:w-8 mt-4 mx-1 place-self-center",
+          },
+          value: state.numWrong,
+        },
+      ];
+    });
+
+    /**
+     * progress value (0-100) to be passed to the Scorecard component
+     */
+    const scorecardProgress = computed(() => {
+      const totalAttempted = state.numCorrect + state.numWrong;
```

Comment:
Isn't this nothing but the below computed var? `numQuestionsAnswered`

Reply:
correct - good catch